docs/commands/json.merge.md

```diff
@@ -1 +1,115 @@
-Sets or updates the JSON value at a path.
+Merge one or more `value` in a JSON document adding, updating, deleting and/or replacing a array at `path` in `key`
```

Comment:
Please don't use `xxx` in the description of the command.
This line must be identical to what we have in commands.json. We currently have there "Sets or updates the JSON value at a path".

Merge on or more values in a JSON document

Reply:
change the commands.json also and simplify the current version as suggested